core/src/main/kotlin/model/DisplaySourceSet.kt

```diff
@@ -8,12 +8,14 @@ import org.jetbrains.dokka.utilities.SelfRepresentingSingletonSet
 data class DisplaySourceSet(
     val sourceSetIDs: CompositeSourceSetID,
     val name: String,
-    val platform: Platform
+    val platform: Platform,
+    val dependentSourceSetIDs: Set<DokkaSourceSetID> = emptySet()
```

Comment:
not ideal, but I guess this could be useful if someone decides to build a graph of source set dependencies or something